app/src/lib/stores/helpers/tutorial-assessor.ts

```diff
@@ -143,5 +148,19 @@ export class OnboardingTutorialAssessor {
     localStorage.removeItem(skipInstallEditorKey)
     this.prStepComplete = false
     localStorage.removeItem(pullRequestStepCompleteKey)
+    this.tutorialPaused = false
+    localStorage.removeItem(tutorialPausedKey)
+  }
+
+  /** Call when the user pauses the tutorial */
+  public pauseTutorial = () => {
+    this.tutorialPaused = true
+    setBoolean(tutorialPausedKey, this.tutorialPaused)
+  }
+
+  /** Call when the user resumes the tutorial */
+  public resumeTutorial = () => {
```

Comment:
I don't believe this needs to be bound?
```suggestion
  public resumeTutorial() {
```